packages/http-client-csharp/eng/scripts/Get-CadlRanch-Coverage.ps1

```diff
@@ -40,6 +44,21 @@ foreach ($directory in $directories) {
     if ($LASTEXITCODE -ne 0) {
         exit $LASTEXITCODE
     }
+
+    # srv-driven contains two separate specs, for two separate clients. We need to generate both.
+    if ($subPath.Contains('srv-driven')) {
```

Comment:
It seems like every language will have this issue. I wonder if the scenario itself can be updated to make it easier, e.g. what if main.tsp imported old.tsp, and old.tsp had a different namespace?

Reply:
Hmm, I ran into issues trying this locally with tsp complaining about duplicate operations, even with the different namespaces. Also, if we go this route, I think we will need to coordinate with other languages since both python and java [generate these specs separately](https://github.com/search?q=repo%3Amicrosoft%2Ftypespec%20srv-driven&type=code).  Do let me know if you have an alternative that avoids updating our scripts.